core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -58,6 +71,113 @@
         left: 42px;
         position: absolute;
       }
+      @media only screen and (max-width : 600px)
+      {
+        .oppia-collection-player-title-font
+        {
+          font-size: 32px;
+        }
+      }
+      exploration-summary-tile md-card {
```

Comment:
Er, but actually ... why would the blank card and preview card be different in this respect? /cc @markhalpin